recipes/jimtcl/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "jimtcl" %}
+{% set version = "0.82" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/msteveb/jimtcl/archive/refs/tags/{{ version }}.tar.gz
+    sha256: e8af929b815e4d30e54ff116b2b933e56c00a02b9110529d1a58660b2469aea7
+  - path: build.sh  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - make                  # [unix]
+    - pkg-config            # [unix]
+    - m2w64-make            # [win]
+    - {{ compiler('m2w64_cc') }}             # [win]
```

Sorry, this is the correct one
```suggestion
    - {{ compiler('m2w64-toolchain') }}             # [win]
```